lib/Mariadb.ts

```diff
@@ -1,12 +1,11 @@
-import mariadb from "mariadb";
-import { NewsItem } from "./News";
+// import mariadb from "mariadb";
```

Comment:
So does this remove the dependency on Mariabdb? Feel like there needs to be a discussion on this. Did the functionality move somewhere? Are we removing the news portion of the website (I'm in favor of that as I don't think it's maintainable in it's current state. especially if we hand it off).

Reply:
I believe we agreed this needed to be commented back in for this PR and a separate PR will be made later to remove mariadb